vc/credential.go

```diff
@@ -0,0 +1,236 @@
+package vc
```

Comment:
I think it'd be helpful to include package level documentation ([reference](https://talks.golang.org/2013/bestpractices.slide#15)). Can also use the `doc.go` convention

Reply:
opened https://github.com/TBD54566975/did-sdk/issues/40